### DISCUSSION TITLE
Deleted a Repo - Repo Not Found

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21785] [New to GitHub]
<p>Hello there,<br>
I had accidentally pushed the wrong folder in a wrong repo, and in vain to delete the wrong folder, I deleted the whole repo (via GitHub).<br>
Now every time I try to push new files on my new repo (via Terminal), I get this message:</p>
<pre><code>fatal: repository 'https://github.com/myusername/myfolder.git/' not found
</code></pre>
<p>Can you please help me to resolve this issue, please?<br>
Thank you!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/nepzydoc">@nepzydoc</a>,</p>
<p>Thank you for being here! Yes, you can restore those repositories yourself by following the directions at:</p>
<p><a href="https://help.github.com/en/github/administering-a-repository/restoring-a-deleted-repository" rel="nofollow noopener">https://help.github.com/en/github/administering-a-repository/restoring-a-deleted-repository</a></p>
<p>It can take up to an hour for a deleted repository to be available for restoration.</p>

**Comment by ghost** [comment]
<p>Thank you <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>  :pray:t3::hugs:</p>
